db/changes.go

```diff
@@ -631,6 +631,9 @@ func (db *Database) SimpleMultiChangesFeed(chans base.Set, options ChangesOption
 		var channelsSince channels.TimedSet
 		if db.user != nil {
 			channelsSince = db.user.FilterToAvailableChannels(chans)
+			if !chans.Contains("*") && len(channelsSince.AsSet()) < len(chans) {
+				base.WarnfCtx(db.Ctx, "Channels restricted from %s to %s", chans.String(), channelsSince.AsSet().String())
```

Comment:
I think perhaps the channel names should be redacted in the logging (at least we've done this in other places.
```suggestion
				base.WarnfCtx(db.Ctx, "Channels restricted from %s to %s", base.UD(chans.String()), base.UD(channelsSince.AsSet().String()))
```

Reply:
@JRascagneres These days, that'll tag each element individually, right? Also, am I right in saying that the `AsSet()` will make the elements identical across both slices by stripping the `:123` from `foo:123` as normally appears in channelsSince type contexts?

Reply:
Calling base.UD() on the stringified is going to redact the set.  We'd want to channel names individually (for cross-referencing) which I think would be something like ```base.UD(channelsSince.AllChannels())```.